bindings/src/main/scala/chrome/contextMenus/bindings/ContextMenus.scala

```diff
@@ -0,0 +1,114 @@
+package chrome.contextMenus.bindings
+
+import chrome.tabs.bindings.Tab
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.{JSName, ScalaJSDefined}
+import scala.scalajs.js.|
+import scala.scalajs.js.JSConverters._
+
+object MenuContexts {
+
+  val ALL = "all"
+  val PAGE = "page"
+  val FRAME = "frame"
+  val SELECTION = "selection"
+  val LINKE = "link"
+  val EDITABLE = "editable"
+  val IMAGE = "image"
+  val VIDEO = "video"
+  val AUDIO = "audio"
+  val LAUNCHER = "launcher"
+  val BROWSER_ACTION = "browser_action"
+  val PAGE_ACTION = "page_action"
+}
+
+@js.native
+@JSName("chrome.contextMenus")
+object ContextMenus extends js.Object{
+
+  def create(createProperties: CreateProperties): String | Int = js.native
+
+  def update(id: String | Int, properties: UpdateProperties): Unit = js.native
+
+  def remove(menuItemId: String | Int, callback: js.Function0[Unit]): String | Int = js.native
+
+  def removeAll(callback: js.Function0[Unit]): Unit = js.native
+
+}
+
+@js.native
+@JSName("chrome.contextMenus.onClicked")
+object OnMenuClicked extends js.Object {
+  def addListener(callback: js.Function2[MenuInfo, Tab, Unit]): Unit = js.native
+}
+
+@ScalaJSDefined
+class UpdateProperties(
+                        val `type`: String = "normal", //"normal", "checkbox", "radio", or "separator"
+                        val title: String,
+                        val checked: js.UndefOr[Boolean] = js.undefined,
+                        val contexts: js.Array[String] = js.Array(MenuContexts.ALL),
```

Comment:
i have not checked all the properties but as a general rule of thump when something is marked `Optional` in the chrome documentation i make it `js.UndefOr` with the default value `js.undefined` to preserve the same behavior as the original js api.